From 162821952e14010531a89bc172fa00f95dcc2c79 Mon Sep 17 00:00:00 2001
From: =?UTF-8?q?Ball=C3=B3=20Gy=C3=B6rgy?= <ballogyor@gmail.com>
Date: Wed, 8 Jul 2020 10:46:34 +0200
Subject: [PATCH] Create output directory if not exist

Instead of trying to create the source directory, which must be already exist.
---
 src/Controllers/ConverterController.vala | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/src/Controllers/ConverterController.vala b/src/Controllers/ConverterController.vala
index b639d11..d05681d 100644
--- a/src/Controllers/ConverterController.vala
+++ b/src/Controllers/ConverterController.vala
@@ -251,7 +251,7 @@ namespace Ciano.Controllers {
          */
         private void start_conversion_process (ItemConversion item, string name_format) {
             try {
-                var directory = File.new_for_path (item.directory);
+                var directory = File.new_for_path (this.settings.output_folder);
                 if (!directory.query_exists ()) {
                     directory.make_directory_with_parents();
                 }
-- 
2.27.0

